pkgs/data/themes/nordic/default.nix

```diff
@@ -8,72 +8,72 @@
 , jdupes
 }:
 
-stdenv.mkDerivation rec {
+stdenvNoCC.mkDerivation rec {
   pname = "nordic";
-  version = "unstable-2023-10-17";
+  version = "unstable-2024-01-20";
```

Comment:
according to https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

version should be `2.2.0-unstable-2024-01-20`

Reply:
Done.

Reply:
this is first time I am interacting with you, very much impressed with your contribution to nixpkgs :heart: